web/client/components/geostory/common/enhancers/withPopupSupport.jsx

```diff
@@ -16,8 +16,6 @@ import {
     withHandlers
 } from 'recompose';
 import uuidv1 from 'uuid/v1';
-import buffer from "turf-buffer";
```

Comment:
this seems that was only used here, so it can be removed from package.json as well

Reply:
done